package/freescale-imx/firmware-imx/Config.in

```diff
@@ -54,21 +54,33 @@ config BR2_PACKAGE_FIRMWARE_IMX_NEEDS_DDR_FW
 	default y if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MM
 	default y if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MN
 	default y if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MP
+	default y if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX93
 
 if BR2_PACKAGE_FIRMWARE_IMX_NEEDS_DDR_FW
 
 choice
-	bool "DDR training binaries"
-	default BR2_PACKAGE_FIRMWARE_IMX_LPDDR4
+	bool "DDR firmware binaries"
+	default BR2_PACKAGE_FIRMWARE_IMX_LPDDR4_PMU_TRAIN if BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8M
```

Comment:
they renamed this upstream?

Reply:
No, it's not renamed in upstream.
There a different DDR binaries:
IMX8M, IMX8MM, IMX8MN, IMX8MP:  lpddr4_pmu_train_?d_?mem_${DDR_FIRMWARE_VERSION}.bin
IMX93: lpddr4_imem_1d_v202201.bin

To me the easiest thing was to create a new type of DDR firmware. As the existing firmware files have prefix `lpddr4_pmu_train` and the new once for IMX93 only `lpddr4`

This rename won't be backward compatible.

Reply:
alright - not sure about the details - just please talk to @bobwolff-cpi and @aswathgajendran-cpi to see if this is any concern on the older platforms.

Reply:
I went back to have only one LPDDR4 choice but added a BR2_PACKAGE_FIRMWARE_IMX_LPDDR4_FW_NAME option which defaults to "lpddr4_pmu_train" which is the prevision firmware filename prefix.

Reply:
I've not yet dug into the buildroot proper repo but only to the emb-platform/os/buildroot/ area for controls over u-boot configs/targets. I'm happy to test something out build-wise, but need some details of what's desired. And if I need a CPCM to do said test, I'll need hardware.